src/primitives.es6

```diff
@@ -33,7 +42,7 @@ export class ElementType {
   toRefract(options = {}) {
     const attributes = this.convertAttributesToRefract('toRefract');
     const initial = {
-      element: this.element,
+      element: this.elementName(),
```

Comment:
I'm :+1: for getters when we don't expect chaining, e.g. for the element name, meta, attributes, and content. Is there any reason **not** to use a getter?